### ISSUE TITLE
Sector & Industry

### ISSUE THREAD

**Issue by lars670** [iexg/IEX-API#105]
## Summary
You provide Sector and Industry but lack the Sector code.

## Details
Example for symbol CMCSA you provide sector: **Consumer Cyclical** and industry: **Entertainment** but the actual numerical code for CMCSA is not provided. The lack of this makes it quite hard for us to place the symbol into the hierachy of sectors. For both GICS and NAICS this is normally a 4-level hierachy. This are to be used for drill-down in a Pie-chart and summarize ones sector exposure.

In short, the numerical code for a companys sector are missing.

### Date/time of issue
Nov. 11. 2017

### API URL
https://api.iextrading.com/1.0/stock/CMCSA/company

### What you expected to happen
To receive the numerical sector code for whatever classification your are using and receiving from your 3:rd parties

### What actually happened
Receiving only Sector and Industry.

### Other Information (sample code, links, etc.)
Which official classification are used?

**Comment by leedstyh** [comment]
I like zacks.com's classification, it's more reasonable I think.

**Comment by lars670** [comment]
Zacks also use a 3 level numerical hieracy. 
So whatever the classification IEX has chosen the code is missing, and that would be nice to have

**Comment by ModelsRUs** [comment]
I agree this would be useful. Without the code, roll-ups are very difficult.

**Comment by lars670** [comment]
Marked as "In development"
Pending this development could you tell which classification is presently used?

**Comment by jkotchoff** [comment]
The Sector and Industry that you publish should match those published by the NASDAQ

ie.
https://www.nasdaq.com/screening/companies-by-name.aspx?letter=0&exchange=nasdaq&render=download
https://www.nasdaq.com/screening/companies-by-name.aspx?letter=0&exchange=nyse&render=download
https://www.nasdaq.com/screening/companies-by-name.aspx?letter=0&exchange=amex&render=download

For example,

https://api.iextrading.com/1.0/stock/eb/company
returns a blank industry and sector

https://api.iextrading.com/1.0/stock/xxii/company
returns an industry of "_Biotechnology_" and sector of "_Healthcare_" (the amex url above lists this as "__Consumer Non-Durables__" and "__Farming/Seeds/Milling__")

https://api.iextrading.com/1.0/stock/aapl/company
returns an industry of "_Computer Hardware_" and sector of "_Technology_" (the nasdaq url above lists this as "__Technology__" and  "__Computer Manufacturing__")

Additionally, it would be useful if you also published the IPO year from the above data sources and provided an exchange_code in the response (eg. with a value of 'NASDAQ'/'NYSE'/'NYSEMKT') instead of the exchange name (such as "Nasdaq Global Select").

**Comment by joshuablackburn-iex** [comment]
@jkotchoff Our current values come from a third party which we are changing for version 2.  We will have better data in the next version. The exchange code will also include IEX.

**Comment by larspeh** [comment]
Rising this question again, and hoping for better than in version 1.

1. What hierachy is used with the new provider? is this a secret?
2. If a hierachy is used; can we see this, or better embed this in the API as a dictionary.
3. Does a numerical code exist for the sector/industry/subsector?

**Comment by pgm111** [comment]
What is the latest update concerning the possible use of GICS for Sector, Industry Group, Industry, and Sub-Industry?

**Comment by evancarter-iex** [comment]
@pgm111 Our primary data partner uses a proprietary sector classification. Their sector classification is used by all major institutions globally. GICS is the most common sector classification, however, S&P charges considerable license fees to use GICS sector codes. We believe the majority of Cloud users are able to use the classifications provided. We do offer a sector performance endpoint (https://iexcloud.io/docs/api/#sector-performance) that uses SPDR ETFs for the main GICS sectors.

**Comment by KirilNN** [comment]
At least you can include information about which SPDR ETF you are using for reference, since there is not way to get performance for the past year for example.

**Comment by ItzhakHengel** [comment]
Tried classifying to GICS using the tags, industry and sector info provided but in some cases the tags are completely off. 
For example NFLX is tagged  "Consumer Services" , "Cable/Satellite TV" with sector "Real Estate and Rental and Leasing".

**Comment by timpaine-iex** [comment]
@KirilNN since that's a separate question (related more to docs I guess), would you mind putting it in a separate issue? It should be relatively easy to back out the sector into the corresponding select sector spdr.

**Comment by timpaine-iex** [comment]
@ItzhakHengel just note that the data provider has shifted since Dec 1, 2020, so this thread will become a little disjointed. I agree that sector for NFLX looks way off (though tags and industry seem ok), and will circle back about that.

**Comment by timpaine-iex** [comment]
Also please note, as far GICS is concerned:
"The Information may not be used to create derivative works or to verify or correct other data or information. For example (but without limitation), the Information may not be used to create indexes, databases, risk models, analytics, software, or in connection with the issuing, offering, sponsoring, managing or marketing of any securities, portfolios, financial products or other investment vehicles utilizing or based on, linked to, tracking or otherwise derived from the Information or any other MSCI data, information, products or services."

**Comment by timpaine-iex** [comment]
closing as stale
